packages/paste-core/components/chat-composer/src/styles.css

```diff
@@ -0,0 +1,8 @@
+.editor-paragraph {
```

Comment:
thought: these styles risk clashing with other app styles once imported. Any change we can rename the class names in the library and then style these as `paste-chat-composer-editor-paragraph` or similar?

Reply:
yeah! I can rename the classes